http_server.go

```diff
@@ -80,9 +82,10 @@ type HTTPServerConfig struct {
 
 func DefaultHTTPServerConfig() *HTTPServerConfig {
 	return &HTTPServerConfig{
-		Protocol:    HTTPScheme,
-		Addr:        ":8080",
-		ReadTimeout: 5 * time.Second,
+		Protocol:     HTTPScheme,
+		Addr:         ":8080",
+		ReadTimeout:  5 * time.Second,
```

Comment:
This is wrong, we should be using read header with some high value like 1m by default.